deployment/windows/build-jellyfin.ps1

```diff
@@ -8,6 +8,7 @@ param(
     [switch]$GenerateZip,
     [string]$InstallLocation = "./dist/jellyfin-win-nsis",
     [string]$UXLocation = "../jellyfin-ux",
+    [string]$InstallTrayApp,
```

Comment:
This should probably be `switch`. Since the script now fails with 
```
D:\a\1\s\deployment\windows\build-jellyfin.ps1 : Missing an argument for parameter 'InstallTrayApp'. Specify a parameter of type 'System.String' and try again.
```

Reply:
Whoops 😬